lib/ziti-tunnel/include/ziti/netif_driver.h

```diff
@@ -17,6 +17,7 @@ typedef int (*uv_poll_req_fn)(netif_handle dev, uv_loop_t *loop, uv_poll_t *tun_
 typedef int (*setup_packet_cb)(netif_handle dev, uv_loop_t *loop, packet_cb cb, void *netif);
 typedef int (*add_route_cb)(netif_handle dev, const char *dest);
 typedef int (*delete_route_cb)(netif_handle dev, const char *dest);
+typedef int (*exclude_route_fn)(netif_handle dev, uv_loop_t *loop, const char *dest);
```

Comment:
What do you think about renaming this to `add_address_cb`? I think it better reflects what the function actually does (rather than why it is being done), and we need to call the same function for hosted connections that set source IP.

Reply:
never mind - I see route_exclusion is different from adding an address. I'll add another cb for add_address.